node/_tools/suites/parallel/test-child-process-execfile.js

```diff
@@ -94,9 +93,10 @@ const denoOptions = [
     assert.strictEqual(err.name, 'AbortError');
     assert.strictEqual(err.signal, undefined);
   });
-  execFile(process.execPath, [echoFixture, 0], { signal }, check);
+  execFile(process.execPath, [...denoOptions, echoFixture, 0], { signal }, check);
 }
 
+/* TODO(Nautigsam) This test requires to check the type but isn't this already covered by tsc?
```

Comment:
> isn't this already covered by tsc?

That's true when user import it in typescript context, but node.js generally doesn't assume the user use typescript. So they check most parameter types at runtime